backend/Makefile

```diff
@@ -17,6 +17,8 @@ backend:
 run:
 	./aro-hcp-backend --location ${LOCATION} \
 		--clusters-service-url http://localhost:8000
+		--cosmos-name ${DB_NAME} \
+		--cosmos-url ${DB_URL}
```

Comment:
where are these two env vars coming from? is the user setting them when running the cmd?

Reply:
From envars, as the deploy make target. I just notice I need to reuse how the DB_URL is get.

Reply:
ack. we tried to not have heavy lookup calls like this one outside of targets. even if it is duplication, i suggest to have the DB_URL lookup in the deploy and run separately